### DISCUSSION TITLE
Joint Agents and DEI Committee Discussion - Changing names of institutions

### DISCUSSION THREAD

**Discussion by ewommack** [ArctosDB/arctos#6648] [Team Posts]
The Agents and DEI Committee should discuss how to support the preservation and changing of institution names that Arctos Agents.

A good example has come up, the [Puget Sound Natural History Collection](https://arctos.database.museum/agent/21277149). 

The following information came through the NHCOL listserv announcing the name change:
Dear Museum community,

The name of the University of Puget Sound's Slater Museum of Natural History has reverted to its original name, the Puget Sound Museum of Natural History. After a multi-year process, the President recommended this change to the Board of Trustees who approved it at their recent meeting. The University’s press release can be found [here](https://www.pugetsound.edu/stories/university-puget-sound-remove-name-former-professor-museum-natural-history) and a Seattle Times story about the renaming, [here](https://www.seattletimes.com/seattle-news/university-of-puget-sound-to-remove-name-of-eugenics-professor-from-museum/).

James Slater, whose collections started the museum, also taught eugenics into the 1950s. The university determined that Slater's long history of teaching eugenics at Puget Sound did not align with the university's mission and values. Please be patient as we work to change the name on our various accounts and in biodiversity portals. Please direct questions or comments to me ([phwimberger@pugetsound.edu](mailto:phwimberger@pugetsound.edu)), or see you at SPNHC!

Thanks,
Peter Wimberger

Director, Puget Sound Museum of Natural History
University of Puget Sound, Tacoma, Washington, USA
_______________________________________________
Nhcoll-l mailing list
[Nhcoll-l@mailman.yale.edu](mailto:Nhcoll-l@mailman.yale.edu)

**Comment by mkoo** [comment]
Arctos has additional dates and details in the agent metadata
